presets/4.3/rc_link/defaults.txt

```diff
@@ -1,77 +1,13 @@
-#$ TITLE: Default 4.3 RC link settings
+#$ TITLE: Reset RC link-feedforward related settings
 #$ FIRMWARE_VERSION: 4.3
 #$ CATEGORY: RC_LINK
 #$ STATUS: OFFICIAL
 #$ KEYWORDS: defaults, reset, rc link, rc_link
 #$ AUTHOR: Betaflight
-#$ DESCRIPTION: Resets all RC Link related parameters to 4.3 defaults
-# Warning: Resets settings for ALL protocols
-# Note: Does NOT reset serial port settings or Rx Link mode (TAER / AETR etc)
+#$ DESCRIPTION: Resets RC_LINK feedforward smoothing/jitter/averaging parameters and RX provider protocol.
 
 set serialrx_provider = SPEK1024
```

Comment:
Pretty much every one of yours and Zak RC_LINK presets are setting this value, so I think if you OK with setting them in the RC_LINK presets, you should be ok also with it in the defaults file.

Reply:
Yes, in an RC_LINK preset, there is a need to set the serial protocol correctly, so that the serial data stream is interpreted correctly.

However, there is no need to 'undo' this with a default, beforehand, because every RC_Link preset will always set this, or the link won't work.  It won't matter what the value is beforehand, it will need to be over-written.

Let's imagine a user who had FrSky and upgraded to ELRS.  The ELRS RC_Link Preset will force the serial provider to CRSF.  It doesn't matter if that Preset loaded a default which converted their prior FrSky serial provider to Spek1024, it will be changed to CRSF.  

The only time a default to Spek1024 would make any difference would be if we had a Spek1024 Preset.  That Preset, and only that Preset, would not need to change serial Rx provider.  

Spek1024 is an archaic Rx protocol, and frankly it makes no sense that it remains our default firmware protocol.  We would be best off setting the firmware default to a more common protocol like CRSF, especially as ELRS gains traction.  

However, it really is not necessary to set this in the reset file, and this line should be removed.

Reply:
ok i can agree here :)